ui/components/CheckboxActions.tsx

```diff
@@ -80,7 +80,7 @@ const DefaultSuspend: React.FC<{
 };
 
 export type Action = {
-  element: React.ReactElement;
+  element: React.FC;
```

Comment:
The prop name probably needs to be `component=` instead of `element` if we want a component instead of an instantiated element.